system/security/LdapSecurity/ldapsecurity.ipp

```diff
@@ -443,6 +444,12 @@ public:
     {
         return m_passwordExpirationWarningDays;
     }
+
+    virtual bool getCheckViewPermissions()
```

Comment:
A function defined within a class definition is implicitly an inline function, so I don't think it is necessary to add "inline" keyword. 

(Although I can still add it, but then I have to change all other functions in same way for consistency...)